src/containers/post/post.component.jsx

```diff
@@ -1,29 +1,65 @@
-import React from "react";
+import React, { useState } from "react";
 import { voteForOption } from "database/votePost";
+import { useSelector } from "react-redux";
 
 const Post = (props) => {
+  const [voteView, setVoteView] = useState(false);
+  const [voteACount, setVoteACount] = useState(props.optionA.length);
+  const [voteBCount, setVoteBCount] = useState(props.optionB.length);
+  const userIDFromState = useSelector((state) => state.users.user.id);
+
+  const handleVote = (obj) => {
+    setVoteView(true);
+    voteForOption(obj, userIDFromState);
```

Comment:
Again `obj` needs to be a more meaningful name. Also good to pass in objects as parameters such as 
```
voteForOption({ meaningfulName1: 5, meaningfulName2: 10}) so that it is more readable
```

A good example of this is how the `handleVote` function is being called in this same file

Reply:
I'll stab at that

Reply:
done